### PR TITLE
at91: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
DEVICE_TITLE is split up into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT
Uses DEVICE_ALT* variables for alternative vendor/retailer names.

### CODE REVIEW

**Comment by mwarning** [comment]
@adrianschmutzler

File: target/linux/at91/image/sama5.mk [hunk 1]
```diff
@@ -54,23 +54,26 @@ endef
 
 define Device/at91-sama5d2_xplained
   $(Device/evaluation-dtb)
-  DEVICE_TITLE := Microchip(Atmel AT91) SAMA5D2 Xplained
+  DEVICE_VENDOR := Microchip(Atmel AT91)
```
Comment (adrianschmutzler) [inline target/linux/at91/image/sama5.mk]: I wonder whether we can't find a nicer solution here.
Either remove the "Atmel AT91" entirely, as the user will have selected at91 explicitly in make menuconfig anyway, or set up alternate names (which I'm not sure about yet).

Comment (mwarning) [reply target/linux/at91/image/sama5.mk]: fine with me

File: target/linux/at91/image/sama5.mk [hunk 2]
```diff
@@ -54,23 +54,26 @@ endef
...
   KERNEL_SIZE := 6144k
   $(Device/evaluation-sdimage)
 endef
 TARGET_DEVICES += at91-sama5d27_som1_ek
 
 define Device/at91-sama5d2_ptc_ek
   $(Device/evaluation-dtb)
-  DEVICE_TITLE := Microchip(Atmel AT91) SAMA5D2 PTC Ek
+  DEVICE_VENDOR := Microchip
+  DEVICE_MODEL := SAMA5D2 PTC EK
```
Comment (adrianschmutzler) [inline target/linux/at91/image/sama5.mk]: Upper-case EK by intention? Didn't do that for the previous device ...

**Comment by adrianschmutzler** [comment]
You should mention that this is the second patch on at91, as you already patched half it.
Maybe one could even add a "remaining" in the commit title.

File: target/linux/at91/image/sama5.mk (hunk 2 above)
Comment (mwarning) [reply target/linux/at91/image/sama5.mk]: it's a typo

**Comment by mwarning** [comment]
@adrianschmutzler done

**Review by adschm — APPROVED** [review]
Looks good and should be quick to review.

**Comment by mwarning** [comment]
@claudyus please review - thanks!

**Comment by hauke** [comment]
Thank you for your patch, it was applied to master.

**Pull request closed without merging**